includes/Core/Assets/Assets.php

```diff
@@ -761,6 +761,22 @@ private function get_inline_base_data() {
 			'isMultisite'       => is_multisite(),
 		);
 
+		if ( Feature_Flags::enabled( 'adsPax' ) ) {
+			$inline_data['plugins'] = array(
+				'wooCommerce'          => array(
+					'installed' => false,
+					'active'    => false,
+				),
+				'googleForWooCommerce' => array(
+					'installed'    => false,
+					'active'       => false,
+					'adsConnected' => false,
+				),
+			);
+
+			$inline_data = $this->update_plugins_inline_data( $inline_data );
```

Comment:
These defaults are redundant and passing the inline data value into the `update_plugins_inline_data` method is not how we do things here. We need to rework this to be consistent with the approach we use for other things. See `get_post_types` for example.

P.S.: this also means we need to rename `update_plugins_inline_data` to `get_plugins_status`.